### DISCUSSION TITLE
Changes in schedules after the competition?

### DISCUSSION THREAD

**Discussion by snoop2head** [fwillett/speechBCI#17] [General]
Dear Brain-to-text benchmark staff members,

Thank you for hosting the competition for past year. 

I recently noticed that schedule after the competition has changed, because I thought ICML 2024 Workshop is mentioned in the [challenges overview page](https://eval.ai/web/challenges/challenge-page/2099/overview) and it seems like it is gone now. Is this a change of schedule or have I seen something wrong?

Thank you.
Sincerely, Young Jin Ahn.

**Comment by fwillett** [comment]
Hi Young,

Unfortunately our plans fell through for ICML 2024, apologies about that. I still plan to organize something online where we can all talk to each other about what we've learned after the competition end date, and hopefully a white paper if there are enough interested submitters. How has the competition been going for you?

Cheers,
Frank
